README.rst

```diff
@@ -0,0 +1,181 @@
+OpenTracing API for Python
```

Comment:
The only change other than formatting was mentioning `upload_docs` in the release step:
```
python setup.py sdist upload -r pypi upload_docs -r pypi
```

If you run `make docs` before doing the upload & upload docs commands the sphinx documentation will be published to https://pythonhosted.org/opentracing.  This makes I possible for other package offers to link directly to the generated documentation for reference.

I'm working on an implementation of opentracing over the Tornado async framework targeting a zip kin backend.  (https://github.com/dave-shawley/sprockets-tracing). I was surprised that the opentracing python API docs weren't published in any of the common places (e.g., readthedocs.io or pythonhosted.org), hence the PR.

Thank you very much.